packages/@ourworldindata/utils/src/Util.ts

```diff
@@ -2000,3 +2000,15 @@ export function getPaginationPageNumbers(
 
     return pageNumbers
 }
+
+/**
+ * Convert 0 to "0th", 1 to "1st", etc.
+ * */
+export function getOrdinalNumberString(n: number): string {
```

Comment:
Not a big deal, but consider using [Intl.PluralRules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules#formatting_text_using_the_returned_tag), which would make this more readable (IMHO) and standardized.

Reply:
@rakyi Just a drive by comment to say thanks for mentioning this, I didn't know about it and this seems pretty neat. Maybe post about it on slack in developers or tools-and-workflows?